src/ensembl/src/content/app/browser/track-panel/track-panel-list/TrackPanelList.tsx

```diff
@@ -18,7 +18,7 @@ import React from 'react';
 import { connect } from 'react-redux';
 import get from 'lodash/get';
 
-import { toggleDrawer, changeDrawerView } from '../../drawer/drawerActions';
+import { toggleDrawer } from '../../drawer/drawerActions';
```

Comment:
`toggleDrawer` isn't used in `TrackPanelList` either 😄

Reply:
Removed it 👍